cli/index.js

```diff
@@ -39,12 +39,15 @@ function upgrade() {
   // read the project's package.json and get currently installed version
   // of the 'dredd-hooks-template' package
   const packageDataPath = path.join(PROJECT_DIR, 'package.json');
-  const packageData = JSON.parse(fs.readFileSync(packageDataPath, { encoding: 'utf-8' }));
+  const packageData = JSON.parse(fs.readFileSync(packageDataPath, 'utf-8'));
```

Comment:
Shouldn't it be `utf8` (without a dash)?

Reply:
Yes, utf-8 I think comes from Python.

utf8 in Node, does utf-8 work though? or spotty test coverage? https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings

Reply:
I thought it doesn't matter 😄 Not sure if in Python, Node.js, or both, but somewhere it doesn't matter between `utf-8`, `utf8`, `UTF-8`, etc. for sure.

Reply:
Added e15641d to address this